src/data/markdown/docs/02 javascript api/08 k6-http/10-haed- url- -params- -.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'head( url, [params] )'
+description: 'Issue an HTTP HEAD request.'
+excerpt: 'Issue an HTTP HEAD request.'
+---
+
+Make a HEAD request.
+
+| Parameter | Type | Description |
+| --------- | ---- | ----------- |
+| url | string / [HTTP URL](/javascript-api/k6-http/url-url#returns) | Request URL (e.g. `http://example.com`). |
+| params (optional) | object | [Params](/javascript-api/k6-http/params) object containing additional request parameters. |
+
+### Returns
+
+| Type                                         | Description           |
+| -------------------------------------------- | --------------------- |
+| [Response](/javascript-api/k6-http/response) | HTTP Response object. |
+
+### Example fetching a URL
+
+<CodeGroup labels={[]}>
+
+```javascript
+import http from 'k6/http';
+
+export default function () {
+  const res = http.head('https://k6.io');
```

Comment:
Maybe it's better to use the `https://test.k6.io` as the target :thinking:

Reply:
Sure, good catch. It was a copy and paste from GET so I have to change its doc also

Reply:
:+1: I'll add then the URL to the release notes.

And also there is a list of all available methods that we should update 
https://mdr-ci.staging.k6.io/docs/refs/pull/588/merge/using-k6/http-requests/#available-methods